job-templates/bridge.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       terminationGracePeriodSeconds: 1800
       containers:
       - name: buildkite-agent
-        image: buildkite/agent:latest
+        image: us.gcr.io/sigma-1330/buildkite/base:1.0
```

Comment:
How was the version "1.0" added? Couldn't find how the dockerfiles/buildkite/base/build.sh was called.